test/e2e/framework.go

```diff
@@ -301,6 +302,69 @@ func (data *TestData) deployAntreaIPSec() error {
 	return data.deployAntreaCommon(antreaIPSecYML, "")
 }
 
+// deployAntreaFlowExporter deploys Antrea with flow exporter config params enabled.
+func (data *TestData) deployAntreaFlowExporter(ipfixCollector string) error {
+	// May be better to change this from configmap rather than directly changing antrea manifest?
```

Comment:
yes please update the ConfigMap instead, it's not as messy

but honestly we need to address https://github.com/vmware-tanzu/antrea/issues/893 asap. The way this is now we never disable the flow exporter once this function is called. So if this test runs first, it's basically the same as just enabling the flow exporter by default for all e2e tests.

Reply:
done